src/protocol.rs

```diff
@@ -12,7 +12,7 @@ use tracing::*;
 pub struct Protocol {
     pub origin: String,
     pub timestamp: u64,
-    message: MAVLinkV2MessageRaw,
+    pub message: MAVLinkV2MessageRaw,
```

Comment:
I believe this is not necessary, as we are implementing both deref and deref mut here